src/id3.js

```diff
@@ -59,13 +59,18 @@ ID3.clearAll = function() {
 ID3.loadTags = function(url, cb, options) {
     options = options || {};
     var dataReader = options["dataReader"] || BufferedBinaryAjax;
+    var onError = options["onError"];

```

Comment:
I should be using the same variable instead of this `}, options["onError"]);` 😕  sorry 😕